From 4892b9e1031fa04ceaab78b6425fdf3d8b89b267 Mon Sep 17 00:00:00 2001
From: "mark.gross@intel.com" <mark.gross@intel.com>
Date: Sat, 14 Feb 2015 07:08:24 -0800
Subject: [PATCH 1/2] fix-missing-sys_rename2 because google is using headers
 from 3.16 in AOSP the back ported the sys calls for renamate2 (not rename2)  
 But they did a sloppy job on x86.

This is a hack to deal with that while we are on 3.14.
Remove this patch when moving past 3.16 for the baseline.
---
 arch/x86/syscalls/syscall_32.tbl |    2 +-
 arch/x86/syscalls/syscall_64.tbl |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

Index: b/arch/x86/syscalls/syscall_32.tbl
===================================================================
--- a/arch/x86/syscalls/syscall_32.tbl	2015-11-16 10:57:48.335071048 -0800
+++ b/arch/x86/syscalls/syscall_32.tbl	2015-11-16 10:58:42.519184973 -0800
@@ -359,5 +359,5 @@
 350	i386	finit_module		sys_finit_module
 351	i386	sched_setattr		sys_sched_setattr
 352	i386	sched_getattr		sys_sched_getattr
-353	i386	renameat2
+#353	i386	renameat2		sys_renameat2
 354	i386	seccomp			sys_seccomp
Index: b/arch/x86/syscalls/syscall_64.tbl
===================================================================
--- a/arch/x86/syscalls/syscall_64.tbl	2015-11-16 10:57:48.335071048 -0800
+++ b/arch/x86/syscalls/syscall_64.tbl	2015-11-16 10:58:22.451139632 -0800
@@ -322,7 +322,7 @@
 313	common	finit_module		sys_finit_module
 314	common	sched_setattr		sys_sched_setattr
 315	common	sched_getattr		sys_sched_getattr
-316	common	renameat2
+#316	common	renameat2		sys_renameat2
 317	common	seccomp			sys_seccomp
 
 #
